src/components/Carousel.tsx

```diff
@@ -47,7 +50,7 @@ const Carousel: React.FC<Props> = ({
   resizeMode = "cover",
   dotColor,
   theme: { colors, spacing },
-  style = { height: 250 },
+  style = { height: screenWidth * 0.5 },
```

Comment:
is there a reason the default can't be a height of 250?

Reply:
When it was hard-coded to 250, it looked weird on large width devices (tablets or pc for web), remember the issue with aspect ratio. This way it will look more or less the same aspect ratio on all devices.